src/handlers/handler.ts

```diff
@@ -11,3 +11,15 @@ export const helloWorld: Handler = (event: APIGatewayEvent, context: Context, ca
 
   callback(null, response);
 }
+
+export const helloUser: Handler = (event: any, context: Context, callback: Callback) => {
+  const response = {
+    statusCode: 200,
+    body: JSON.stringify({
+      message: `Hello ${event.name}!`,
```

Comment:
I'm actually after being able to POST the name to this endpoint using a REST client.  The results from a test post look like this:
`{
  "message": "Hello undefined!",
  "input": {
    "resource": "/hello-user",
    "path": "/hello-user",
    "httpMethod": "POST",
    "headers": {
      "Accept": "*/*",
      "Accept-Encoding": "gzip, deflate, br",
      "Accept-Language": "en-US,en;q=0.5",
      "CloudFront-Forwarded-Proto": "https",
      "CloudFront-Is-Desktop-Viewer": "true",
      "CloudFront-Is-Mobile-Viewer": "false",
      "CloudFront-Is-SmartTV-Viewer": "false",
      "CloudFront-Is-Tablet-Viewer": "false",
      "CloudFront-Viewer-Country": "US",
      "content-type": "text/plain;charset=UTF-8",
      "dnt": "1",
      "Host": "hc9pdzn4kc.execute-api.us-east-1.amazonaws.com",
      "origin": "moz-extension://913b8cf5-56c5-4a45-8e9b-87c73c41b967",
      "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:57.0) Gecko/20100101 Firefox/57.0",
      "Via": "2.0 0576b942ae9f4fc9c0b62b0736e9bfd6.cloudfront.net (CloudFront)",
      "X-Amz-Cf-Id": "0OJoIuyglBJWL_wSp_8560yfLOYSbLrqlOjYJYHu1yMiCPucen76hg==",
      "X-Amzn-Trace-Id": "Root=1-5a6d4292-0660e3651a88d4783087d500",
      "X-Forwarded-For": "73.93.143.9, 54.239.203.107",
      "X-Forwarded-Port": "443",
      "X-Forwarded-Proto": "https"
    },
    "queryStringParameters": null,
    "pathParameters": null,
    "stageVariables": null,
    "requestContext": {
      "requestTime": "28/Jan/2018:03:25:06 +0000",
      "path": "/dev/hello-user",
      "accountId": "078302068589",
      "protocol": "HTTP/1.1",
      "resourceId": "zhlh0t",
      "stage": "dev",
      "requestTimeEpoch": 1517109906383,
      "requestId": "d60e85f0-03da-11e8-b5e8-2952b3373c49",
      "identity": {
        "cognitoIdentityPoolId": null,
        "accountId": null,
        "cognitoIdentityId": null,
        "caller": null,
        "sourceIp": "73.93.143.9",
        "accessKey": null,
        "cognitoAuthenticationType": null,
        "cognitoAuthenticationProvider": null,
        "userArn": null,
        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:57.0) Gecko/20100101 Firefox/57.0",
        "user": null
      },
      "resourcePath": "/hello-user",
      "httpMethod": "POST",
      "apiId": "hc9pdzn4kc"
    },
    "body": "{\"name\":\"Tim\"}",
    "isBase64Encoded": false
  }
}`

So you should be able to use the value from `event.body.name`.